jcstress-wrapper/src/main/kotlin/komem/litmus/JCStressRunner.kt

```diff
@@ -0,0 +1,47 @@
+package komem.litmus
+
+import java.nio.file.Path
+import kotlin.system.exitProcess
+
+/**
+ * Note that this 'runner' is severely different from all others.
+ */
+object JCStressRunner : LitmusRunner() {
+    override fun <S : Any> runTest(params: LitmusRunParams, test: LitmusTest<S>): LitmusResult =
+        throw NotImplementedError("jcstress runner is not supposed to be used like this")
+
+    fun runJCStress(
+        params: LitmusRunParams?,
+        tests: Collection<LitmusTest<*>>,
+        jcstressDirectory: Path,
+        jcstressFreeArgs: List<String>,
```

Comment:
to unify the interfaces, you can, for example, pass additional arguments in the constructor of the class.

Reply:
:ok_hand: